app/index.js

```diff
@@ -46,10 +46,17 @@ module.exports = JhipsterGenerator.extend({
             type: Boolean,
             defaults: true
         });
+        // This method adds support for a `--skip-entities` flag
+        this.option('skip-entities', {
```

Comment:
In my opinion this shouldn't be enabled by default as most novice users wouldn't know this, may be we can ask a question after regenerating the app if they want to regen all entities or the arg option should be true by default with flag as `--regenerate-entities`
I would prefer a question after the write tasks of app files